qiskit/ignis/experiments/calibration/methods/cross_resonance.py

```diff
@@ -0,0 +1,88 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Data source to generate schedule."""
+
+
+from typing import Optional, List
+
+from qiskit.ignis.experiments.calibration import CircuitBasedGenerator
+from qiskit.ignis.experiments.calibration.data_processing import DataProcessingSteps
+from qiskit.ignis.experiments.calibration.cal_base_analysis import BaseCalibrationAnalysis
+from qiskit.ignis.experiments.calibration.instruction_data import InstructionsDefinition
+from qiskit.ignis.experiments.calibration.cal_base_experiment import BaseCalibrationExperiment
+from qiskit.ignis.experiments.calibration.analysis.trigonometric import CosinusoidalFit
+
+
+class RoughCRAmplitude(BaseCalibrationExperiment):
+    """Performs a rough amplitude calibration by scanning the amplitude of the pulse."""
+
+    def __init__(self,
+                 inst_def: InstructionsDefinition,
+                 qc: int,
```

Comment:
Can this be a `Tuple[int, int]` to conform to the definition of `QuantumCircuit`?

Reply:
Sounds good.